views/cook.ejs

```diff
@@ -5,6 +5,7 @@
   <link href="css/style.css" rel="stylesheet">
   <link href="css/cook.css" rel="stylesheet">
   <script src="js/cook.js"></script>
+  <title>LOGIN PAGE</title>
```

Comment:
pisze tutaj bo zmiany o ktorych pisze sa juz sa na masterze i nie moge ich bezposrednio skomentowac
klasa info-order nie powinna nazywac sie order-info skoro macie juz order-status? (linia ok. 19)
czy height img 90em (linia ok. 15) nie powinien byc 1. wrzucowny w style 2. wyrazony za pomoca jakies innej jednostki?

Reply:
Co do nazewnictwa, już P odpisałem... postaram sie ujednolicić w wolnej.... 
1 coś nie działało wtedy, wiec jak zadziałało w html'u to zostawiliśmy, poprawie jak nie zapomnę
2 Jak pisałem wyżej % nie działają, px nie powinno się używać (widłak), a w sumie o vw i vh niepamiętałem wgl